geotrek/tourism/filters.py

```diff
@@ -32,3 +32,11 @@ class Meta(StructureRelatedFilterSet.Meta):
         model = TouristicEvent
         fields = StructureRelatedFilterSet.Meta.fields + [
             'published', 'type', 'themes', 'after', 'before', 'approved']
+
+
+class TouristicEventApiFilterSet(django_filters.FilterSet):
+    ends_after = django_filters.DateFilter('end_date', lookup_type='gte')
+
+    class Meta:
+        model = TouristicEvent
+        fields = ['end_date', ]
```

Comment:
No harm done, besides you made me realize i made a tremendous mistake with my branches, ah too many features at once is not a good idea. ;)